### ISSUE TITLE
Celery Beat run many times Task with Crontab X * * * * minutes

### ISSUE THREAD

**Issue by oortega** [celery/celery#4107]
Hello, When I add a Periodic task with crontab every 3 minutes, this run many times does not respect the 3 minutes.

## Steps to reproduce

Attach an image with the problem:

http://g.recordit.co/F14hwQ1rll.gif

I tried  this but is the same result:

```
from celery.schedules import crontab
app.conf.beat_schedule = {
     'Suma': {
         'task': 'test1.tasks.add',
         'schedule': crontab(minute=4),
         'args': (15, 15),
     },
 }
```

But if task run crontab  every X hours not happened.

## Expected behavior
Run every 3 minutes

## Actual behavior
Run many times every second, 

## Setup
I Installed:
(I installed this version because I nedd this change https://github.com/celery/celery/pull/3890)

pip install https://github.com/celery/celery/zipball/master#egg=celery
 pip install https://github.com/celery/billiard/zipball/master#egg=billiard
 pip install https://github.com/celery/py-amqp/zipball/master#egg=amqp
 pip install https://github.com/celery/kombu/zipball/master#egg=kombu
 pip install https://github.com/celery/vine/zipball/master#egg=vine
 pip install https://github.com/celery/django-celery-beat/zipball/master#egg=django-celery-beat

I run celery:

/var/waps/entornos/dcelery2/bin/celery -A dcelery beat -l info -S django 
/var/waps/entornos/dcelery2/bin/celery worker -A dcelery --loglevel=INFO

#Pip Freeze
amqp==2.1.4
billiard==3.5.0.2
celery==4.0.2
Django==1.11.2
django-celery-beat==1.0.1
django-celery-results==1.0.1
kombu==4.0.2
MySQL-python==1.2.5
pytz==2017.2
vine==1.1.3

CELERY_BROKER_URL = 'redis://localhost:6379'
#CELERY_BROKER_URL = 'amqp://guest:guest@localhost//'

#: Only add pickle to this list if your broker is secured
#: from unwanted access (see userguide/security.html)
CELERY_ACCEPT_CONTENT = ['json']
CELERY_RESULT_BACKEND = 'django-db'
CELERY_TASK_SERIALIZER = 'json'
CELERY_BEAT_SCHEDULER = 'django_celery_beat.schedulers:DatabaseScheduler'
CELERY_ENABLE_UTC=True
CELERY_TIMEZONE = TIME_ZONE

This is my project test: https://bitbucket.org/ortega/dcelery

Someone else happens?

**Comment by RabidCicada** [comment]
I think this is related to celery/django-celery-beat/#57 and that the problem is in the interplay of django-celery-beat and celery.  I had the same behavior when I messed with timezone settings then I forgot to run 

> python manage.py shell
> >>> from djcelery.models import PeriodicTask
> >>> PeriodicTask.objects.update(last_run_at=None)

from the [docs](http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html) when I changed timezone stuff between runs.  Is there a chance you forgot to clear out the last_run_at after changing timezone settings?

It seems this is all inter-related, and I expect you had the same or similar problem as I did.

**Comment by oortega** [comment]
@RabidCicada  I applied PeriodicTask.objects.update (last_run_at = None), but I still have the same problem

Do you want to do a test on my pc?
```
from django_celery_beat.models import  PeriodicTask
PeriodicTask.objects.update(last_run_at=None)
```

**Comment by thedrow** [comment]
Please try upgrading your dependencies. We released Celery, Kombu, amqp and vine.

**Comment by oortega** [comment]
@thedrow  same issue. This is new enviroment:

amqp==2.2.2
billiard==3.5.0.3
celery==4.1.0
Django==1.11.2
django-celery-beat==1.0.1
django-celery-results==1.0.1
kombu==4.1.0
MySQL-python==1.2.5
pytz==2017.2
redis==2.10.6
vine==1.1.4


Some idea?

Regards.

**Comment by amitripshtos** [comment]
Having same issue, same env

**Comment by eillarra** [comment]
I have the same issue with 4.1.0 not respecting the crontab() configuration: it just creates thousands of tasks until you stop the beat.

Periodic tasks work as expected again if I revert to 4.0.2.

**Comment by auvipy** [comment]
is it still exist on 4.2.1 release?

**Comment by oortega** [comment]
@auvipy is solved, many thanks.

@dongshengYoung, @eillarra 

**packages installed:**

amqp==2.3.2
billiard==3.5.0.4
celery==4.2.1
Django==1.11.2
django-celery-beat==1.1.1
django-celery-results==1.0.1
kombu==4.2.1
MySQL-python==1.2.5
pytz==2018.5
redis==2.10.6
vine==1.1.4

**Comment by auvipy** [comment]
thanks!! closing as fixed on celery 4.2.x
